app/src/main/java/com/sikaeapps/to_dolist/AddNewItemActivity.java

```diff
@@ -0,0 +1,52 @@
+package com.sikaeapps.to_dolist;
+
+import android.app.Activity;
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.util.Log;
+import android.view.View;
+import android.widget.EditText;
+import android.widget.Toast;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+public class AddNewItemActivity extends AppCompatActivity {
+
+    private static final int OK_CODE = 123;
+    @BindView(R.id.new_item_title)
+    EditText title;
+
+    @BindView(R.id.new_item_description)
+    EditText description;
+
+    @BindView(R.id.new_item_location)
+    EditText location;
+
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_add_new_item);
+        ButterKnife.bind(this);
+    }
+
+    public void saveNewItem(View view) {
+        ItemManager manager = (ItemManager) getIntent().getSerializableExtra("manager");
+
+        Item newItem = new Item(title.getText().toString(),
+                description.getText().toString(),
+                location.getText().toString());
+
+        try {
+            manager.addItem(newItem);
+            Intent intent = new Intent();
+            intent.putExtra("manager", manager);
+            setResult(Activity.RESULT_OK, intent);
```

Comment:
Create a Class that stores static constants and use this value instead of RESULT_OK and REQUEST_CODE